src/test/java/api/requests/StaffSlipsTests.java

```diff
@@ -761,6 +770,43 @@ void responseContainsPickSlipsForManyTitleLevelRequestsCreatedForSameHoldingAndI
     assertResponseHasItems(response, batchSize + 1, SlipsType.PICK_SLIPS);
   }
 
+  @Test
+  void responseContainsNoRecordsIfPickSlipsDisabled() {
+    circulationSettingFixture.configurePrintEventLogFeature(true);
+
+    UUID servicePointId = servicePointsFixture.cd1().getId();
+    val item = itemsFixture.basedUponSmallAngryPlanet();
+    val james = usersFixture.james();
+
+    RequestBuilder firstRequestBuilder = new RequestBuilder()
+      .withStatus(RequestStatus.OPEN_NOT_YET_FILLED.getValue())
+      .page()
+      .withPickupServicePointId(servicePointId)
+      .forItem(item)
+      .by(james);
+
+    IndividualResource firstRequest = requestsClient.create(firstRequestBuilder);
+    Response response = SlipsType.PICK_SLIPS.get(servicePointId);
+
+    assertThat(response.getStatusCode(), is(HTTP_OK));
+    assertResponseHasItems(response, 1, SlipsType.PICK_SLIPS);
+    assertResponseContains(response, SlipsType.PICK_SLIPS, item, firstRequest, james);
+
+    circulationSettingFixture.configurePrintEventLogFeature(false);
+    response = SlipsType.PICK_SLIPS.get(servicePointId);
+
+    assertThat(response.getStatusCode(), is(HTTP_OK));
+    assertResponseHasItems(response, 0, SlipsType.PICK_SLIPS);
+  }
+
+  private void configureSlipsFeatureForType(SlipsType slipsType, boolean enabled) {
+    if (slipsType == SlipsType.PICK_SLIPS) {
+      circulationSettingFixture.configurePrintEventLogFeature(enabled);
+    } else {
```

Comment:
Just in case we will introduce a new slips type:
```suggestion
    } else if (slipsType == SlipsType.SEARCH_SLIPS) {
```